script/generateHistory.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+from subprocess import PIPE, Popen
+
+adapterPath = 'adapter/'
+aggregatorDefaultPath = 'aggregator/default/'
+aggregatorBaobabPath = 'aggregator/baobab/'
+
+adapters = sorted(os.listdir(adapterPath))
+aggregatorsDefault = sorted(os.listdir(aggregatorDefaultPath))
+aggregatorsBaobab = sorted(os.listdir(aggregatorBaobabPath))
+
+
+def loadJsonFromPath(filePath):
+    with open(filePath) as json_file:
+        data = json.load(json_file)
+    return data
+
+
+def makeLine(words):
+    for word in words:
+        if type(word) == dict:
+            print('| [{}]({}) '.format(word['value'], word['url']), end='')
+        elif type(word) == list:
+            print('| ', end='')
+            for version in word:
+                print('[{}]({})'.format(version['value'], version['url']), end='')
+                if version != word[-1]:
+                    print(', ', end='')
+        else:
+            if word == '':
+                word = '-'
+            print('| {} '.format(word), end='')
+    print(' |')
+
+
+def makeEmptyLine(words):
+    for i in range(len(words)):
```

Comment:
There does not have to be space between `|` and `---`. It could have just been something like this:

```python
def makeHorizontalSeparator(words):
    print("|---" * len(words), end="|\n")
```